cmd/core-data/Dockerfile

```diff
@@ -27,8 +27,6 @@ WORKDIR /edgex-go
 # The main mirrors are giving us timeout issues on builds periodically.
 # So we can try these.
 
-RUN sed -e 's/dl-cdn[.]alpinelinux.org/dl-4.alpinelinux.org/g' -i~ /etc/apk/repositories
-
```

Comment:
also remove the comments

Reply:
yep, did on some, missed on these . Good catch.

Reply:
Removed all left over comments